lte/gateway/python/magma/pipelined/ifaces.py

```diff
@@ -29,3 +29,10 @@ def monitor_ifaces(iface_names):
             status = 1 if iface in active else 0
             NETWORK_IFACE_STATUS.labels(iface_name=iface).set(status)
         yield from asyncio.sleep(POLL_INTERVAL_SECONDS)
+
+
+def get_mac_address(interface_name: str) -> str:
+    if_addresses = netifaces.ifaddresses(interface_name)[netifaces.AF_LINK]
```

Comment:
This line can rise `ValueError: You must specify a valid interface name.`, e.g., for a non existing interface name. I think this is good. Just asking the question if this should be combined with the value error below.

Reply:
I think it's fine to return a ValueError with different message for both cases.